src/facade/command_id.h

```diff
@@ -34,27 +34,27 @@ class CommandId {
   CommandId(const char* name, uint32_t mask, int8_t arity, int8_t first_key, int8_t last_key,
             uint32_t acl_categories);
 
-  std::string_view name() const {
```

Comment:
I wish you asked first
https://google.github.io/styleguide/cppguide.html#Function_Names
```
Accessors and mutators (get and set functions) may be named like variables. These often correspond to actual member variables, but this is not required. For example, int count() and void set_count(int count).
```

Reply:
It says *may* which as I understand either way is fine. IMO I do find it confusing to have some functions names like that,and in other places even for accessors or seters to not use lower_case. It's inconsistent.

I would like this merged as I am leaning towards consistency over exceptions but if you strongly object I will close this.

Also no worries, it didn't take much time, vim search and replace is very efficient in doing this kind of changes so it's not like I lost time.

Anyway it's your call! 

(Typed this from my phone excuse my handwriting)

Reply:
Also on the other hand, Abseil containers follow this conventions 🤷‍♂️🤷‍♂️

Reply:
This PR de facto changes the styleguide guidelines from "may" to "should not". I do not have the strong opinion about either option because I accept the styleguide as is.  Having said that, I am against changing the styleguide via a PR being sent because a team member feels the styleguide is wrong. Styleguide is a status quo. If you want to discuss it - our Wednesday meetings are for that. And frankly there are better options to improve the code like refactoring to reduce the LOC.